src/pages/HomePage.vue

```diff
@@ -28,73 +43,122 @@
 import MainTitle from "../components/MainTitle.vue";
 import BaseButton from '../components/BaseButton.vue';
 import ScreeningsHeader from '../components/ScreeningsHeader.vue';
+import ScreeningsList from '../components/ScreeningsList.vue';
+import MovieCard from '../components/MovieCard.vue';
+import ContactInfo from '../components/ContactInfo.vue';
+import photo1 from "../assets/predator.png";
+import photo2 from "../assets/commando.png";
+import photo3 from "../assets/terminator.png";
+
 
 export default {
   name: "HomePage",
   components: {
     MainTitle,
     BaseButton,
     ScreeningsHeader,
+    ScreeningsList,
+    MovieCard,
+    ContactInfo,
+    
   },
 
+  data(){
+    return {
+      movies: [
+        {id: '1a', title: 'Predator', length: 90, poster_url: photo1, genre: {id: 1, name: 'Action'}} ,
```

Comment:
It is probably not finished now, but in the future, we want to replace this mock data with a real one from the API.

Reply:
Yes, it's not finished. I want get those items directly from api.